test/runtime.js

```diff
@@ -19,7 +26,75 @@
 /* TODO: images */
 /* TODO: math */
 /* TODO: random */
-/* TODO: vectors */
+
+/* vectors */
+
+var iota = 0.0001;
+function equalish(a,b){
```

Comment:
Oh good. I'll use yours because mine seems to be hiding some genuine errors that it should have caught.